### PR TITLE
Removing spread operator

### PR DESCRIPTION
**Pull request by krogh**
Hi thanks for the good work
Spread operator doesnt seem to work without babel.

### CODE REVIEW

File: tasks/prettier.js [hunk 1]
```diff
@@ -86,10 +86,9 @@ function prettierTask(grunt) {
         // If f.dest is undefined, then write formatted code to original files.
         codeFiles.map(function(filepath) {
           unformattedCode = grunt.file.read(filepath);
-          formattedCode = prettier.format(unformattedCode, {
-            ...options,
-            parser: getParser(filepath, options.parser)
-          });
+          formattedCode = prettier.format(unformattedCode, 
+            Object.assign({ parser: getParser(filepath, options.parser) }, options)
```
Comment (poalrom) [inline tasks/prettier.js]: Your version not override parser in 'options' object. Line should be like this:
```js
Object.assign({}, options, { parser: getParser(filepath, options.parser) })
```

File: tasks/prettier.js [hunk 2]
```diff
@@ -99,10 +98,9 @@ function prettierTask(grunt) {
           return grunt.file.read(filepath);
         });
 
-        formattedCode = prettier.format(unformattedCode.join(''), {
-          ...options,
-          parser: getParser(codeFiles[0], options.parser)
-        });
+        formattedCode = prettier.format(unformattedCode.join(''),
+          Object.assign({ parser: getParser(codeFiles[0], options.parser) }, options)
```
Comment (poalrom) [inline tasks/prettier.js]: Same error like above

**Review by poalrom — COMMENTED** [review]
Spread operator supports in node v5+. But you can fix errors from review and I will merge pull request

**Comment by krogh** [comment]
hi sorry for the delay. Was on x-mas vacation without computer.
We run node in version 8.1.0
Maybe the the issue is related to this : https://stackoverflow.com/questions/40066731/node-v6-failing-on-object-spread

I did as you instructed. When i run 'npm run test' I still get test errors.
Thanks

**Comment by poalrom** [comment]
Thanks for pull request, I merge it.
My test not fails on last repo version

**Pull request merged**